app/models/asset_field_type.rb

```diff
@@ -52,6 +53,29 @@ def mapping
 
   private
 
+  def is_image(mime_type)
```

Comment:
The difference is in `image?` compares against `asset_content_type` which is not yet defined when I need it

```ruby
def image?
  asset_content_type =~ %r{^(image|(x-)?application)/(bmp|gif|jpeg|jpg|pjpeg|png|x-png)$}
end
```

also `image?` returns either `nil` or `0`  and I hoping for a true boolean method